backend/config/urls.py

```diff
@@ -55,6 +55,7 @@ def edx_path(route, component_name):
     # React views
     path('', render_react_view, {'component_name': 'IndexView'}),
     path('all_view/', render_react_view, {'component_name': 'AllView'}),
+    path('adventure/', render_react_view, {'component_name': 'ChooseAdventureView'}),
```

Comment:
use the `edx_path` helper function here, so that your thing shows up in fake edxview